source/intel.js

```diff
@@ -574,18 +574,37 @@ function NeptunesPrideAgent() {
     let arrival = new Date(now.getTime() + msplus);
     let p = prefix !== undefined ? prefix : "ETA ";
     let ttt = p + ampm(arrival.getHours(), arrival.getMinutes());
-    if (arrival.getDay() != now.getDay())
-      ttt = `${p}${days[arrival.getDay()]} @ ${ampm(
-        arrival.getHours(),
-        arrival.getMinutes(),
-      )}`;
+    if (!NeptunesPride.gameConfig.turnBased) {
+      ttt = p + ampm(arrival.getHours(), arrival.getMinutes());
+      if (arrival.getDay() != now.getDay())
+        ttt =
+          p +
+          days[arrival.getDay()] +
+          " @ " +
+          ampm(arrival.getHours(), arrival.getMinutes());
+    } else {
+      totalETA = arrival - now;
+      ttt = p + Crux.formatTime(totalETA);
+    }
     return ttt;
   };
   let tickToEtaString = function (tick, prefix) {
     let msplus = msToTick(tick);
     return msToEtaString(msplus, prefix);
   };
-
+  let msToCycleString = function (msplus, prefix) {
+    let p = prefix !== undefined ? prefix : "ETA";
+    let cycleLength = NeptunesPride.universe.galaxy.production_rate;
+    let tickLength = NeptunesPride.universe.galaxy.tick_rate;
+    let ticksToComplete = Math.ceil(msplus / 60000 / tickLength);
+    let ttt =
```

Comment:
Please try to name things with words. This type of naming is unreadable especially without a comment.